bold.py

```diff
@@ -6,4 +7,50 @@ class BoldBlock(Block):
     '''
     def _render(self):
         return '<strong>{}</strong>'.format(self.content())
-      
\ No newline at end of file
+
+    @staticmethod
+    def parse(content):
+        '''
+        Input: a text content
+        Output: a tuple (int, int, BoldBlock)
+            [0]: start index,
+            [1]: end index,
+            [2]: created class,
+
+        Note, the index of start and end follow the convention of [start, end)
+        '''
+        pattern = r'([*]+)(.*?)([*]+)'
```

Comment:
> 
> 
> One things you miss here:
> 
>     * what if a block is *_a_*? or **a**? should also consider escape case.

```*_a_*``` will not match, since only 1 asterisk would not satisfy the if conditions.
```**a**``` will be bold `a`, it's covered in the test ```**test**``` -> `test`

Reply:
How about the double underscore case? It's also valid for Bold. `__This will also be bold__`

Reply:
Yup, would like to discuss with you guys that, are we going to support all alternative ways?